rules/manager.go

```diff
@@ -267,7 +268,7 @@ func NewGroup(name, file string, interval time.Duration, rules []Rule, shouldRes
 func (g *Group) Name() string { return g.name }
 
 // File returns the group's file.
-func (g *Group) File() string { return g.file }
+func (g *Group) File() string { return filepath.Base(g.file) }
```

Comment:
I think it's ok to preserve the full path here (this function is already used by `/rules` as well, where it already shows the whole path, see e.g. http://demo.robustperception.io:9090/rules).